QWeb/keywords/input_.py

```diff
@@ -25,13 +25,21 @@
 from selenium.webdriver.remote.webelement import WebElement
 from robot.api import logger
 from robot.api.deco import keyword
+try:
+    from robot.api.types import Secret
+    BUILT_IN_SECRET = True
+except ImportError:
+    BUILT_IN_SECRET = False
+    Secret = str
```

Comment:
Feel questionable to define the `Secret` type as `str`

I'd prefer defining `str_or_secret` in the try-except and then flip the `isinstance(input_text, Secret)` check

```
try:
    from robot.api.types import Secret
    BUILT_IN_SECRET = True
    str_or_secret = Union[str, Secret]
except ImportError:
    BUILT_IN_SECRET = False
    str_or_secret = str
```

Then on L290 
`if BUILT_IN_SECRET and not isinstance(input_text, str)`

Reply:
Yes, I tried all sorts of ways and I'm pretty sure mypy didn't like this. But I'll check again.

Reply:
I just realized that the `BUILT_IN_SECRET` bool is unnnecessary if the L290 check is inverted as suggested in the thread start

`if not isinstance(input_text, str)`